doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,29 @@
 
 <h3>New features since last release</h3>
 
+* Measurement statistics can now be collected for mid-circuit measurements. Currently,
```

Comment:
Is this to be updated?

Reply:
This branch has `mcm-stats-1` (branch for #4544 ) as its base, this is just a merge artifact. Should be fixed now